webpack.config.common.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+module.exports = {
+  entry: {
+    app: './src/index.js',
+  },
+plugins: [
+    new CleanWebpackPlugin(['app']),
+    new HtmlWebpackPlugin({
+      title: 'React-trainings',
+      filename: 'index.html',
+      template: './dist/index.html',
+    }),
+  ],
+module: {
+    rules: [
+      {
+        test: /\.js$/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: [
```

Comment:
It's unnecessary lines, because it will exclude using of another presets and plugins from the `.babelrc` file.
In other words it's something like filter :-)